pkg/kubelet/server/server.go

```diff
@@ -145,7 +146,7 @@ func ListenAndServeKubeletServer(
 	enableDebuggingHandlers,
 	enableContentionProfiling,
 	enableSystemLogHandler bool) {
-	klog.Infof("Starting to listen on %s:%d", address, port)
```

Comment:
That's will be result in `Starting to listen on 127.0.0.1=8080`

Reply:
Sorry, I don't understand your mean. `Starting to listen on 127.0.0.1=8080` is right? @yuzhiquan

Reply:
127.0.0.1:8080 more properly.

Reply:
ok. I understand , I will change the code. thanks very much. My english is poor.